### PR TITLE
Ways of working - Minimal documentation standards

### PR DESCRIPTION
**Pull request by jeff-horton-ho-sas**

### CODE REVIEW

**Review by carmeloMdho — DISMISSED** [review]

File: docs/standards/minimal-documentation-set-for-a-product.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- Documentation
+- Ways of working
+---
+
+{% import "_includes/macros/tags.njk" as pageTag %}
+
+Last updated: {{ page.date | postDate }}
+{{ pageTag.tags(tags)  }}
+
+A Home Office engineering product should be documented to a minimum standard so that new engineers can be easily
```
Comment (robertdeniszczyc2) [inline docs/standards/minimal-documentation-set-for-a-product.md:17]:
```suggestion
A product should be documented to a minimum standard so that new engineers can be easily 
```

Given this is a standard for Home Office Software Engineering on the HO SEGAS site, we can probably reword this to drop the "Home Office engineering" bit.

File: docs/standards/minimal-documentation-set-for-a-product.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - Security audits
+    - Service design package
+
+### Product documentation MUST include key architectural views
+
+The architectural views may contain architecture diagrams, security models, entity-relationship models. The views should allow an engineer to understand how the components interact at a high and low level.
+
+### Product documentation MUST include a decision log
+
+A decision log should record key design decisions in context. A pattern should be selected to compliment the team ways of working, and may include for example [Architecture Decision Records](https://adr.github.io/).
```
Comment (robertdeniszczyc2) [inline docs/standards/minimal-documentation-set-for-a-product.md:51]:
```suggestion
A decision log should record key design decisions in context. A pattern should be selected to complement the team ways of working, and may include for example [Architecture Decision Records](https://adr.github.io/). 
```

File: docs/standards/minimal-documentation-set-for-a-product.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+A decision log should record key design decisions in context. A pattern should be selected to compliment the team ways of working, and may include for example [Architecture Decision Records](https://adr.github.io/). 
+
+### Product documentation MUST include incident management information
+
+An incident management process should include technical points of contact, runbooks, and disaster recovery information.
+
+### Product documentation MUST include information about observability
+
+Documentation should include details of logging, monitoring, and alerting designs, and links to alerting and monitoring dashboards.
```
Comment (robertdeniszczyc2) [inline docs/standards/minimal-documentation-set-for-a-product.md:59]:
```suggestion
Documentation should include details of logging, monitoring and alerting designs, and links to alerting and monitoring dashboards.
```

**Review by robertdeniszczyc2 — APPROVED** [review]

**Review by carmeloMdho — APPROVED** [review]
